### DISCUSSION TITLE
I have question about version

### DISCUSSION THREAD

**Discussion by fenil-arcticgrey** [EvgeniyMukhamedjanov/liquid-ajax-cart#101] [General]
Hello everyone!

I am working with Ajax cart I have the `liquid-ajax-cart-v1.10.1.js` version in my theme. But past few days some of the functionality has stopped working. Is anything has changed in the `liquid-ajax-cart-v1.10.1.js` version recently?

**Comment by EvgeniyMukhamedjanov** [comment]
Hello @fenil-arcticgrey ,
Nothing was changed in liquid-ajax-cart-v1.10.1.
The issue might come from Shopify as it is changing its API constantly or new apps installed.
Could you explain what exactly got broken and provide a link to the theme for me to review.

**Comment by fenil-arcticgrey** [comment]
Hello @EvgeniyMukhamedjanov!

Thank you for responding quickly.

We have several functionalities in the cart drawer. One of these has stopped working across all themes, and we haven't made any code changes recently.

Here's our theme preview: https://thecupstore.com/products/sport-beer-mug-25oz/?&preview_theme_id=142066450654

Add any product to the cart, and you'll see an event calendar at the bottom of the cart drawer [https://prnt.sc/1iEov7EY9Br0]. This calendar adds a product based on the selected date [https://prnt.sc/DBTSJiTdxP-y]. We are having issues with this feature. When selecting a date that has a $100 or $50 charge, it doesn't work. You can open the cart drawer by hovering over it with your cursor. Here's a visual reference showing the issue.

Visual reference: 
https://www.awesomescreenshot.com/video/28725080?key=c6fd78564d6b21667b5489d2a7507f99

Thanks!

**Reply by EvgeniyMukhamedjanov** [reply]
Hello,
I see that liquid-ajax-cart itself is working, as items are added, removed without page reload.
The issue is with autodeleting rush-100 and rush-50 items.
This logic for autodeleting is defined somewhere in JavaScript code of the page.
I suppose this is the one:
![image](https://github.com/EvgeniyMukhamedjanov/liquid-ajax-cart/assets/13817355/47dce5ac-2d8c-47fa-9160-a45df138d7fb)

I'm not sure why the code is written to remove the rush-100 and rush-50 items, the developer who built that should be aware.
